components/brave_rewards/browser/test/rewards_flag_browsertest.cc

```diff
@@ -76,6 +81,8 @@ class RewardsFlagBrowserTest : public InProcessBrowserTest {
   }
 
   void TearDown() override {
+    brave_rewards::RewardsFlags::SetForceParsingForTesting(false);
```

Comment:
I think that this should be called from TearDownOnMainThread(). We can rename TearDown() to TearDownOnMainThread() as there is no other logic there.

Reply:
Fixed :-)